app/code/Magento/SimpleProductGraphQl/composer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "magento/module-simple-product-graph-ql",
```

Comment:
This should be part of `CatalogGraphql` module since simple product type is defined in `Catalog` module.